lib/tdf3/src/version.ts

```diff
@@ -1,2 +1,2 @@
-export const version = '0.2.0';
+export const version = '4.3.0';
```

Comment:
The `version` field is set automatically by the `bump-version.sh` script, so rename this to either tdfSpecVersion or targetFileFormatVersion or similar

```suggestion
export const version = '0.2.0';
export const tdfSpecVersion = '4.3.0';
```

Reply:
I've deleted this file to consolidate on lib/src/version.ts